tools/test/audio/std_utils/stdnotch_get.m

```diff
@@ -43,7 +43,7 @@
 
 target_q = 2.1;
 bw = (2*fn/fs)/target_q;
-if exist('iirnotch.m') == 2
+if exist(fullfile('.', 'iirnotch.m'),'file') == 2
```

Comment:
Looks like the commit subject overflowed and subject should be lowercase. Can you copy the message fix below.
```
audio : tools :  fix matlab/octave directory compatibility

To support exist function - fullfile() path addition is to prevent Matlab (differently than Octave)
to give false positive with some other directory location with it's search path

Added correction to isempty.

Signed-off-by: sriram <malladi.sastry@intel.com>
```

Reply:
done

Reply:
Have you pushed ? I can only see the old version ?

Reply:
@ShriramShastry sorry, still seeing the old version ? fwiw ```git rebase -i``` can be used to fix this commit message, just mark it for reword(r) and edit the message and then you will need to force push otherwise the update wont be visible here.